sw/device/tests/BUILD

```diff
@@ -129,6 +130,45 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "alert_handler_lpg_clkoff_test",
+    srcs = ["alert_handler_lpg_clkoff_test.c"],
+    cw310 = cw310_params(timeout = "long"),
+    targets = [
```

Comment:
that sounds good to me.

Reply:
I believe the CI currently only runs specific Verilator tests, so adding a Verilator target shouldn't have any effect on CI: https://github.com/lowRISC/opentitan/blob/7aa862374f40d29ec7dd2029b20aec650c1f79b0/ci/scripts/run-verilator-tests.sh

You can always add a really long timeout parameter (e.g. `"eternal"`) to make it clear that the test takes a while to run. The CW310 tests are currently filtered based on timeout -- which means, actually, that this test with `timeout="long"` will not be run in CI even on FPGA: https://github.com/lowRISC/opentitan/blob/7aa862374f40d29ec7dd2029b20aec650c1f79b0/ci/scripts/run-fpga-cw310-tests.sh#L42

Leaving it as non-Verilator seems fine to me, but I thought I'd clear up these points since I happened to look into this recently for other reasons.